ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -177,3 +180,10 @@ private fun SelectPackageButton(
         Text(text = "Purchase ${aPackage.identifier}. Price: ${aPackage.product.price.formatted}")
     }
 }
+
+@Preview(showBackground = true)
+@Composable
+fun Template2PaywallPreview() {
+    InternalPaywallView(offering = TestData.template2Offering)
```

Comment:
This doesn't actually work, and I have no idea why. Preview fails to load:
```
java.lang.ClassNotFoundException: com.revenuecat.purchases.Package   at java.lang.ClassLoader.loadClass  at java.lang.ClassLoader.loadClass  at java.lang.Class.getDeclaredMethods0  at java.lang.Class.privateGetDeclaredMethods  at java.lang.Class.getDeclaredMethods  at androidx.compose.ui.tooling.ComposableInvoker.getDeclaredCompatibleMethod   ... (ComposableInvoker.kt:54)   at androidx.compose.ui.tooling.ComposableInvoker.findComposableMethod(ComposableInvoker.kt:74)
```

Reply:
Hmm I checked out this branch and I'm getting:
<img width="309" alt="image" src="https://github.com/RevenueCat/purchases-android/assets/808417/ca4bc97f-f775-41c1-bf9e-f4bc5a49f7e8">
Which seems to make sense, since we don't have packages with those ids.

I wasn't able to reproduce your issue. Happy to chat today to figure this out!

Reply:
I fixed the data now so all packages are found. But now I'm running into this from `PurchaseButton`:
> java.lang.IllegalStateException: Error finding activity